plume-common/src/activity_pub/request.rs

```diff
@@ -41,6 +41,10 @@ impl Digest {
         }
     }
 
+    pub fn verify_header(&self, other: &Digest) -> bool {
+        self.value()==other.value()
+    }
+
     pub fn algorithm(&self) -> &str {
```

Comment:
i just realised we have a lot of public functions here, but don't document any

Reply:
Yes, we should do it. Also we should probably use https://git.asonix.dog/asonix/http-signatures instead of rolling our own code